federatedscope/autotune/fts/utils.py

```diff
@@ -0,0 +1,523 @@
+from datetime import datetime
+import GPy
+import pickle
+import time
+import numpy as np
+import warnings
+
+from scipydirect import direct
+
+warnings.filterwarnings('ignore')
+
+
```

Comment:
Code in this file seems to be largely borrowed from FTS's repo, right? If it is the case, please provide declaration at the beginning of this file and their copyright in /LICENSE.